plugins/index.js

```diff
@@ -150,6 +150,7 @@ export function removePlugin(manifest) {
     }
     unregisterAllPluginWebSocketEvents(manifest.id);
     unregisterPluginReconnectHandler(manifest.id);
+    new PluginRegistry(manifest.id).unregisterAdminConsolePlugin();
```

Comment:
I think it's really odd that we have to create a `PluginRegister` to invoke `unregisterAdminConsolePlugin` action on it. Can we create an encapsulating action that's invoked by `unregisterAdminConsolePlugin` and you can invoke it here directly.